spec/unit/spotlight/dor/indexer_spec.rb

```diff
@@ -214,7 +214,9 @@
       </mods>
     EOF
   end
-  describe "#add_folder_name" do
+
+  # rubocop:disable Metrics/LineLength
```

Comment:
I think 10 is too small for a method length cutoff.  I think 15 would be fine, or even 20.